client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content/hooks/test/use-row-metadata.ts

```diff
@@ -56,8 +55,16 @@ const FAKE_SITE: Site = {
 		mobile: 10,
 		desktop: 10,
 	},
-	php_version_num: 7.4,
+	php_version_num: '7.4',
```

Comment:
The `php_version_num` field is a number returned by the endpoint

Reply:
Reverted back to number in fb88090.